src/components/desktop/desktop.vue

```diff
@@ -11,7 +11,9 @@
         class="x-list x-list--vertical x-list--gap-05 x-list--align-stretch x-list__item--expand"
       >
         <SearchBox />
-        <RelatedTags v-if="!$x.isEmpathizeOpen && $x.relatedTags.length > 0" />
+        <LocationProvider location="predictive_layer">
```

Comment:
Should the search-box be also contained in the predictive layer? Right now it sends `none` as location.

Reply:
I have no idea, we changed the locations definitions too many times. Let's leave it as `none` at the moment. We can change it in whenever is necessary